### PR TITLE
Fix base voltage config loading

### PR DESCRIPTION
**Pull request by geofjamg**
Signed-off-by: Geoffroy Jamgotchian <geoffroy.jamgotchian@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Regression

**What is the current behavior?** *(You can also link to an open issue here)*
We don't have anymore the jar resource fallback when config is not overloaded by user in its $HOME/.itools.

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: commons/src/main/java/com/powsybl/commons/config/BaseVoltagesConfig.java [hunk 1]
```diff
@@ -66,12 +69,23 @@ public static BaseVoltagesConfig fromInputStream(InputStream configInputStream)
...
+        Objects.requireNonNull(configFileName);
+        Path configFile = configDir.resolve(configFileName);
+        if (Files.exists(configFile)) {
+            try (InputStream configInputStream = Files.newInputStream(configFile)) {
+                return fromInputStream(configInputStream);
+            } catch (IOException e) {
+                throw new UncheckedIOException(e);
+            }
+        } else {
+            InputStream configInputStream = BaseVoltagesConfig.class.getResourceAsStream("/" + configFileName);
```
Comment (flo-dup) [inline commons/src/main/java/com/powsybl/commons/config/BaseVoltagesConfig.java:83]: Could you add a log to tell the user the default config file has been loaded?

File: commons/src/main/java/com/powsybl/commons/config/BaseVoltagesConfig.java [hunk 2]
```diff
@@ -66,12 +69,23 @@ public static BaseVoltagesConfig fromInputStream(InputStream configInputStream)
...
+            if (configInputStream != null) {
+                return fromInputStream(configInputStream);
+            } else {
+                throw new PowsyblException("No base voltages configuration found");
```
Comment (flo-dup) [inline commons/src/main/java/com/powsybl/commons/config/BaseVoltagesConfig.java:87]: Maybe add default: `"No default base voltages configurations found"`

**Review by flo-dup — APPROVED** [review]

File: commons/src/main/java/com/powsybl/commons/config/BaseVoltagesConfig.java (hunk 1 above)
Comment (geofjamg) [reply commons/src/main/java/com/powsybl/commons/config/BaseVoltagesConfig.java:89]: Done

File: commons/src/main/java/com/powsybl/commons/config/BaseVoltagesConfig.java (hunk 2 above)
Comment (geofjamg) [reply commons/src/main/java/com/powsybl/commons/config/BaseVoltagesConfig.java]: Done

**Review by flo-dup — APPROVED** [review]

**Pull request merged**